pyfolio/plotting.py

```diff
@@ -30,18 +30,32 @@
 from . import pos
 
 
-def set_plot_defaults():
-    """
-    Sets default plotting/charting colors/styles.
-    """
+def plotting_context(func):
+    """Decorator to set plotting context during function call."""
+    def call_w_context(*args, **kwargs):
+        set_context = kwargs.pop('set_context', True)
+        if set_context:
+            with context():
+                return func(*args, **kwargs)
+        else:
+            return func(*args, **kwargs)
+    return call_w_context
+
+
+def context(context='notebook', font_scale=1.5, rc=None):
```

Comment:
One way to make this useful would be to replace the `set_context` boolean arg below with a `plotting_context` arg, which will be used if set, and will use the defaults here if `None`.  Then users could construct their preferred plotting context with this function and pass it to the plotting functions.